apis/kueue/v1beta1/provisioningrequestconfig_types.go

```diff
@@ -76,6 +81,13 @@ type ProvisioningRequestConfigSpec struct {
 	//
 	// +optional
 	PodSetUpdates *ProvisioningRequestPodSetUpdates `json:"podSetUpdates,omitempty"`
+
+	// podSetMergePolicy specifies the policy for merging PodSets before being passed
+	// to the cluster autoscaler.
+	//
+	// +optional
+	// +kubebuilder:validation:Enum=Identical;ProvisioningDirectives
+	PodSetMergePolicy *ProvisioningRequestConfigPodSetMergePolicy `json:"podSetMergePolicy,omitempty"`
```

Comment:
@mwielgus @mimowo Do you assume any usecase to addadditional parameters in the future? If yes, we can make a space for now in the following: 
This is the similar decision as admissionChecks and admissionCheckStrategies in ClusterQueue.

```yaml
porSetMergePolicy:
  mode: xxx
```

Reply:
We don't have such use case for now. When they come we can introduce the podSetMergeConfig or podSetMergeStrategy with more details.

Reply:
> We don't have such use case for now. When they come we can introduce the podSetMergeConfig or podSetMergeStrategy with more details.

SGTM, thanks